inspirehep/modules/workflows/api.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from inspire_utils.dedupers import dedupe_list
+
+from inspirehep.modules.workflows.tasks.refextract import (
+    extract_references_from_raw_refs,
+    match_references,
+)
+
+
+def get_linked_references(references):
```

Comment:
hmm I'd put this on utils

Reply:
mmm ok, it can be a util :+1:

Reply:
why not move the whole reference matcher to a different module (`refextract`?)? it's not really related to the workflows (just happens to be used by the workflows only currently).

Reply:
@michamos makes total sense, I'm not sure how easy it is though

Reply:
@michamos @drjova I have moved everything not related to workflows from `workflows/refextract` to `refextract/tasks`. Give it a look and let me know how it looks like :mag_right:

Reply:
I don't think that's the right place for it. `tasks.py` should be reserved for celery tasks (or workflow tasks). This is neither.

Reply:
I agree, but sadly this is not the case. I would put it in `api`, wdyt?

Reply:
``utils.py`` is more suitable IMO ☺️